packages/create-rsbuild/src/index.ts

```diff
@@ -84,7 +84,7 @@ async function main() {
   const distFolder = path.join(cwd, targetDir);
 
   copyFolder(commonFolder, distFolder, version);
-  copyFolder(srcFolder, distFolder, version);
+  copyFolder(srcFolder, distFolder, version, targetDir);
```

Comment:
A folder may have multi-level paths, we can use the basename of the path instead.

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/08293527-b115-4236-97cd-4da124030605)

Reply:
Get ~